projects/plugins/beta/src/admin/admin.js

```diff
@@ -1,4 +1,12 @@
 ( function () {
+	// Avoid running on main plugin selection page.
+	if (
+		'/wp-admin/admin.php?page=jetpack-beta' ===
+		window.location.pathname + window.location.search
```

Comment:
I don't like this check. Look for whether the relevant DOM elements were found instead. Or just don't register it from the PHP in the first place.

Reply:
Thanks for looking Brad, that makes sense. Updated in https://github.com/Automattic/jetpack/pull/20390/commits/cc6cbf64e8d989853b07025af8c9e4930efc41c0